src/TraceEvent/Computers/TraceProcess.cs

```diff
@@ -384,6 +450,7 @@ internal TraceProcess FindProcessAndIndex(int processID, long timeQPC, out int i
         private TraceLog log;
         private TraceEventDispatcher source;
         internal event Action<TraceProcess> OnInitialized;
+        internal event Action<TraceProcess> OnRemoved;
```

Comment:
Lets call this OnProcessStop not on Removed, and go ahead and rename OnInitialized to OnProcessStart.   I realize that this Initialized was probably originally chosen because it is called slightly more places than a process start, but we actually have only one concept (we don't give two kinds of callbacks), and that while slightly inaccurate, calling these 'OnProcessStart' and 'OnProcessStop' will not cause important confusion (the only differences are in weird cases where you don't have a start, and even then, the right thing to do is to make believe you do (which is what happens).  

In short I just want naming to be consistent, and for us to not make fine distinctions unless we really believe it is important for end users to understand the fine distinction.

Reply:
Done.